src/Components/Authentication/Mobile/LoginForm.tsx

```diff
@@ -118,29 +176,42 @@ class MobileLoginFormWithSystemContext extends Component<
         )}
       </Step>,
     ]
+  }
+
+  render() {
+    const { steps, hideBackButton, advanceWizardStep } = this.state
+
     return (
       <Wizard steps={steps} onComplete={this.onSubmit}>
         {context => {
           const {
             wizard,
-            form: { handleSubmit, values, status, isSubmitting },
+            form: { handleSubmit, actions, values, status, isSubmitting },
           } = context
-          const { currentStep, isLastStep } = wizard
+
+          const { currentStep, isLastStep, next } = wizard
+
+          if (advanceWizardStep) {
+            next(values, actions)
+            this.setState({ advanceWizardStep: false })
+          }
```

Comment:
Having a state change within the `render` function doesn't feel right but I'm not sure how to trigger this step advance otherwise.

Reply:
Yeah having this "impure" render definitely feels weird, but I'm not seeing a better solution either.